### PR TITLE
Removing unused models

### PR DESCRIPTION
**Pull request by likhita24**
Removed the below Models as they are unused.

1. HomeEventCard
2. HomeBriefCard
3. HomeImageCarousel

### CODE REVIEW

**Comment by harsh-not-haarsh** [comment]
@likhita24 you were supposed to remove the unused models, not comment them out. Also the PR has some flake8 issues, fix them as well

File: main/admin.py [hunk 1]
```diff
@@ -1,13 +1,5 @@
...
-class HomeImageCarouselAdmin(admin.ModelAdmin):
-    class Meta:
-        model = HomeImageCarousel
-
-
-admin.site.register(HomeImageCarousel, HomeImageCarouselAdmin)
+from .models import NavBarOptions, NavBarSubOptions, OurTeam
 
 
 @admin.register(NavBarOptions)
```
Comment (harsh-not-haarsh) [inline main/admin.py:5]: there are several other classes like the navbar options, can you verify if they are used or not?

Comment (likhita24) [reply main/admin.py:5]: I have observed that NavBarOptions class is being used in main\base.html. If there are any objects present in main_navbaroptions table, they would be fetched by main\context_processors.py file into the variable 'navbaroptions'. So I think this model is required. Please let me know if it's wrong.

Comment (likhita24) [reply main/admin.py:5]: As far as I checked, I didn't find any more unused models.

**Comment by killbotXD** [comment]
@harsh-not-haarsh I think HomeEventCard is necessary I think. it would be used to add dynamic cards to the homepage. when the server issue with the media directory is fixed.

**Comment by harsh-not-haarsh** [comment]
@likhita24 yeah you should not remove the home event card, also you should first discuss the models you are removing on the issue itself.

**Comment by killbotXD** [comment]
> @likhita24 yeah you should not remove the home event card, also you should first discuss the models you are removing on the issue itself.

should we discuss
possibly over a video call?

**Comment by harsh-not-haarsh** [comment]
> > @likhita24 yeah you should not remove the home event card, also you should first discuss the models you are removing on the issue itself.
> 
> should we discuss
> possibly over a video call?

we discussed it over the last scrum meeting if @likhita24 wants we can have another one soon. Let me know.

**Comment by likhita24** [comment]
> > @likhita24 yeah you should not remove the home event card, also you should first discuss the models you are removing on the issue itself.
> 
> should we discuss
> possibly over a video call?

@harsh-not-haarsh I'll make sure that I will discuss with you before I remove any other models. And @killbotXD yeah we can do a video call to discuss about the issue.

**Pull request closed without merging**